src/Media_Command.php

```diff
@@ -183,6 +183,9 @@ function regenerate( $args, $assoc_args = array() ) {
 	 * [--porcelain]
 	 * : Output just the new attachment ID.
 	 *
+	 * [--filetime]
+	 * : Use the file modified time as the post date. Only works if the file is local.
```

Comment:
Is there any reason not to make this the default behavior?

Reply:
It would technically be a breaking change, but it would certainly make sense. Not sure anyone would rely on the old behaviour.

Reply:
The only argument I can think of against making this the default behavior is that it will only work when the files are local.